jdaviz/app.py

```diff
@@ -1796,11 +1796,20 @@ def _reparent_subsets(self, old_parent, new_parent=None):
                     # Translate bounds through WCS if needed
                     if (self.config == "imviz" and
                             self._jdaviz_helper.plugins["Orientation"].link_type == "WCS"):
+
+                        # Default shape for WCS-only layers is 10x10, but it doesn't really matter
+                        # since we only need the angles.
+                        old_angle, _, old_flip = get_compass_info(old_parent.coords, (10, 10))[-3:]
+                        new_angle, _, new_flip = get_compass_info(new_parent.coords, (10, 10))[-3:]
+                        if old_flip != new_flip:
+                            relative_angle = 180 - new_angle - old_angle
```

Comment:
I think this still assumes some symmetry in the subcomponent of the subset and would break if we ever introduce lasso subsets, right?  Maybe just worth a comment so we don't get tripped up by it in the future if we ever add that.

Reply:
FWIW, there is no lasso shape support in regions nor photutils. So it would probably be turned into some polygon that has no concept of angle anyway (though you would have to rotate it properly or whatever before you turn it into polygon), so maybe also put that in comment?